packages/expect-puppeteer/src/matchers/toMatchElement.test.js

```diff
@@ -37,6 +37,38 @@ describe('toMatchElement', () => {
         expect(error.message).toMatch('waiting for function failed')
       }
     })
+
+    it('should match using visible options', async () => {
```

Comment:
Your test is not correct, with try / catch you have to set the number of expectations you expect in the test:

```js
expect.assertions(10)
```